common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -122,7 +122,7 @@ trait ABTestSwitches {
     "ab-sign-in-gate-tertius",
     "Test new sign in component on 2nd article view",
     owners = Seq(Owner.withGithub("coldlink"),Owner.withGithub("dominickendrick")),
-    safeState = Off,
+    safeState = On,
```

Comment:
is this intentional?

Reply:
oh woops, that wasn't accidental, setting it to `On` means I can get the test running locally, but it won't cause any problems as the test is already switched `On` explicitly on production.

Reply:
worth reverting if `Off` is considered the 'safe state'